buildSrc/src/main/kotlin/Config.kt

```diff
@@ -114,14 +114,6 @@ object Libs {
     // https://github.com/airbnb/lottie-android
     const val lottie = "com.airbnb.android:lottie:3.4.0"
 
-    // koin
```

Comment:
can you keep this, it is not dependency

Reply:
Yeah sure, I added the koin dependency urls in the latest commit.